oeps/archived/oep-0037-bp-test-data.rst

```diff
@@ -23,6 +23,12 @@ OEP-37: Dev Data
    * - Review Period
      - 2019-06-19 to 2019-11-25
 
+
+.. attention::
+
+   This proposal is deferred as there is no current plans to actively work on
+   implementing this work.
```

Comment:
Actually this is a pretty high Arch-BOM priority captured in https://github.com/edx/edx-arch-experiments/issues/138 , we're hoping to start work on it in the next 1-3 months.  It also surfaced in https://openedx.atlassian.net/wiki/spaces/AC/pages/3615850497/Development+Environment+Vision#Continue-OEP-37-(Dev-Data)-implementation-%26-refinement as one of the top 3 next steps to improving our dev environments.

Reply:
My preference would be to not move it because:

1. It creates extra work to move it back later
2. The optics of launching a nontrivial implementation project for an archived OEP aren't great
3. Seriously, this is one of the team's top priorities for Q2

But let me know if that poses any problems I'm not aware of.

Reply:
Sounds good, I'll remove that move from this PR.